libraries/bsg_manycore_cuda.cpp

```diff
@@ -2034,6 +2037,11 @@ static int hb_mc_device_tiles_set_config_symbols (hb_mc_device_t *device,
 
 		// Set tile's id __bsg_id symbol.
 		// calculate tile's id 
+		// __bsg_id symbol is the flat index of a tile 
+		// inside its tile group with respect to the origin tile
+		// __bsg_id = __bsg_y * __bsg_tiles_X + __bsg_y
+		// __bsg_tiles_X/Y symbols define the dimensions of manycore
+		// which are set to the tile group dimensions 
```

Comment:
This also doesn't make sense to me. They define the dimensions of [the] manycore, and are also set to the tile group dimensions?

That seems contradictory.